scripts/SasSurveyArtifactNormalizer.psm1

```
@@ -0,0 +1,283 @@
+}
+
+function Get-SasSurveyArtifactId {
+    [CmdletBinding()]
+    param([Parameter(Mandatory = $true)][string]$Path, [Parameter(Mandatory = $true)][string]$Role)
+    $safeBase = [System.IO.Path]::GetFileNameWithoutExtension($Path) -replace '[^A-Za-z0-9._-]', '-'
+    $sha = [System.Security.Cryptography.SHA256]::Create()
+    try { $bytes = [Text.Encoding]::UTF8.GetBytes([IO.Path]::GetFullPath($Path).ToLowerInvariant()); $hash = ([BitConverter]::ToString($sha.ComputeHash($bytes))).Replace('-', '').ToLowerInvariant().Substring(0,8) } finally { $sha.Dispose() }
+    "$Role-$safeBase-$hash"
+}
+
+function Test-SasSurveyDenominatorPackage {
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Test-SasSurveyDenominatorPackage' does not have a help comment.

(PSProvideCommentHelp)